zindi/user.py

```diff
@@ -1,429 +1,443 @@
-# Imports
-
-import sys, os
-
-## To avoid errors of importing before instalation
-currentdir = os.path.dirname(os.path.realpath(__file__))
-parentdir = os.path.dirname(currentdir)
-sys.path.append(parentdir)
```

Comment:
`pip` can install in [editable mode](https://pip.pypa.io/en/stable/cli/pip_install/#editable-installs):

```bash
pip install -e .
```
You can install the package, and still keep editing it.